backend/controllers/prisonerProfile/prisonerLocationHistory.js

```diff
@@ -26,11 +56,26 @@ module.exports = ({ prisonApi, logError }) => async (req, res) => {
     ])
 
     const userCaseLoadIds = userCaseLoads.map(caseLoad => caseLoad.caseLoadId)
-
     const { bookingId, firstName, lastName } = prisonerDetails
-
     const currentPrisonerDetails = locationHistory.find(record => record.bookingId === bookingId) || {}
 
+    const movementMadeByName = currentPrisonerDetails.movementMadeBy
+      ? await fetchStaffName(res.locals, currentPrisonerDetails.movementMadeBy, prisonApi)
```

Comment:
This will throw an exception if it fails. With error containing No staff name found. Is that expected?

Reply:
if it fails it should return `'No staff name found'` I'll write a test for this

Reply:
It will throw a new error when used with await.